docs/instructions.md

```diff
@@ -62,16 +79,94 @@ env = android_env.load(
 )
 ```
 
-## Example scripts
+## Example RL agent scripts
+
+The `examples` directory contains a few simple example agent setups, such as:
+
+*   [`run_random_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_random_agent.py):
+    Runs a simple loop performing randomly selected actions in the environment.
+*   [`run_acme_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_acme_agent.py):
+    Runs a training loop with an [Acme](https://deepmind.com/research/publications/Acme)
+    DQN agent, implemented in the popular DeepMind RL framework. This will
+    require to install the [`acme`](https://github.com/deepmind/acme)
+    dependency.
+*   [`run_human_agent.py`](https://github.com/deepmind/android_env/blob/main/examples/run_human_agent.py):
+    Creates a [`pygame`](https://www.pygame.org) instance that lets a human user
+    interact with the environment and observe environment mechanics, such as
+    rewards or task extras. You will need to install the [PyGame] dependency.
+
+For instance, here is how you can run `run_random_agent.py` in a folder where
+you have your APK file, such as
+the [Apple Flinger](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md#apple-flinger)
+game.
+(You can find the TAR file with the APK and `.textproto` files
+[here](https://storage.googleapis.com/android_env-tasks/apple_flinger.tar.gz)
+and on the
+[example tasks](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md)
+page.)
+
+```shell
+python3 run_random_agent.py \
+--avd_name='my_avd',
+--android_avd_home=/Users/username/.android/avd \
+--android_sdk_root=/Users/username/Library/Android/sdk \
+--emulator_path=/Users/username/Library/Android/sdk/emulator/emulator \
+--adb_path=/Users/username/Library/Android/sdk/platform-tools/adb \
+--num_steps=1000 \
+--task_path=/Users/username/<PATH-TO-APPLE-FLINGER-FILES>/apple_flinger_M_1_1.textproto
+```
+
+## Example with code: a random agent playing Droidfish (chess)
+
+Below is an example of a random agent playing chess in Droidfish using
+`droidfish_black_1.textproto` over 1000 steps (where `1` indicates the
+[level of difficulty](https://github.com/deepmind/android_env/blob/main/docs/example_tasks.md#droidfish)).
+You need to execute the Python code in a folder where you have the Droidfish APK
+file. (Download the Droidfish TAR file with the APK and `.textproto` files
+[here](https://storage.googleapis.com/android_env-tasks/droidfish.tar.gz)
```

Comment:
Same comment as above. It'll be easier to keep all the links to tasks in a single page.

Reply:
Duplicate example removed.